python/ray/_private/ray_logging/constants.py

```diff
@@ -41,6 +41,9 @@ class LogKey(str, Enum):
     NODE_ID = "node_id"
     ACTOR_ID = "actor_id"
     TASK_ID = "task_id"
+    ACTOR_NAME = "actor_name"
+    TASK_NAME = "task_name"
+    TASK_FUNCTION = "task_function"
```

Comment:
Do we need task_function? I thought we only need task_name which should be function name by default?

Reply:
You are right. By default, in general, we the task name as the function name. 

I still see 2 differences between task_name & task_function. That's why I kept them both.
1. It is possible to set a customized task name by specifying `name` in options, but task function will always be the function call name
2. The specific string in task_function will contain the python module name comparing to default task_name will only contain the function itself.

Reply:
@alanwguo do we need function name or task name is enough for now. I feel having task_name is enough.

Reply:
I've seen task name can explode in cardinality but function name is a bit more stable. Having both could be useful where we can decide which to use depending on performance characteristics. For example if task_name is too high cardinality, we can choose to use func name exclusively.

Reply:
ok